docs/docsite/Makefile

```diff
@@ -132,24 +123,40 @@ else
 endif
 
 htmldocs: ansible_structure generate_rst
-	CPUS=$(CPUS) $(MAKE) -f Makefile.sphinx html
+	if $(PYTHON) -c "import sys, packaging.version as p; sys.exit(not p.Version('$(MAJOR_VERSION)') >  p.Version('2.10'))" ; then \
+		CPUS=$(CPUS) $(MAKE) -f Makefile.sphinx 'DOCS_VARIANTS=-t ansible' html; \
+	else \
+		CPUS=$(CPUS) $(MAKE) -f Makefile.sphinx 'DOCS_VARIANTS=-t 2.10' html; \
+	fi
 
 core_htmldocs: core_structure core_generate_rst
-	CPUS=$(CPUS) $(MAKE) -f Makefile.sphinx html
+ifdef LANGOPTS
+	CPUS=$(CPUS) $(MAKE) -f Makefile.sphinx 'DOCS_VARIANTS=-t core_lang' html
+else
+	CPUS=$(CPUS) $(MAKE) -f Makefile.sphinx 'DOCS_VARIANTS=-t core' html
+endif
 
 singlehtmldocs: ansible_structure generate_rst
-	CPUS=$(CPUS) $(MAKE) -f Makefile.sphinx singlehtml
+	if $(PYTHON) -c "import sys, packaging.version as p; sys.exit(not p.Version('$(MAJOR_VERSION)') >  p.Version('2.10'))" ; then \
```

Comment:
I think we can get rid of the 2.10 check by now ;) But let's do that in a follow-up PR.

Reply:
That's what I mentioned earlier FWIW.